docs/source/transaction_family_specifications/track_and_trade_transaction_family.rst

```diff
@@ -0,0 +1,747 @@
+       // Approximately when this proposal was created, as a Unix UTC timestamp
+       uint64 timestamp = 2;
+
+       // The public key of the Agent that created the Proposal
+       string issuing_agent = 3;
+
+       // The public key of the Agent to which the Proposal is addressed
+       string receiving_agent = 4;
+
+       // Whether the Proposal is for transfer of ownership or
+       // custodianship or reporter authorization
+       Role role = 5;
+
+       // The names of properties for which the reporter is being authorized
+       // (empty for owner or custodian transfers)
+       repeated string properties = 6;
+
+       // Whether the Proposal is open, accepted, rejected, or canceled.
+       // For a given Record and receiving Agent, there can be only one
+       // open Proposal at a time for each role.
+       Status status = 7;
+
+       // human-readable terms of transfer
+       string terms = 8;
+   }
+
+
+Proposals with the same address are stored in a list sorted
+alphabetically first by record_id, then by receiving_agent, then by
+timestamp (earliest to latest).
+
+.. code-block:: protobuf
+
+   message ProposalContainer {
+       repeated Proposal entries = 1;
+   }
+
+.. _Addressing:
+
+Addressing
+----------
+
+T&T objects are stored under the namespace obtained by taking the
+first six characters of the SHA-512 hash of the string
+``track_and_trade``:
+
+.. code-block:: pycon
+
+   >>> def get_hash(string):
+   ...     return hashlib.sha512(string.encode('utf-8')).hexdigest()
+   ...
+   >>> get_hash('track_and_trade')[:6]
+   '1c1108'
+
+After its namespace prefix, the next two characters of a T&T object's
+address are a string based on the object's type:
+
+- Agent: ``ae``
+- Property / PropertyPage: ``ea``
+- Proposal: ``aa``
+- Record: ``ec``
+- Record Type: ``ee``
+
+The remaining 62 characters of an object's address are determined by
+its type:
+
+- Agent: the first 62 characters of the hash of its public key.
+- Property: the first 36 characters of the hash of the identifier of
+  its associated Record plus the first 22 characters of the hash of
+  its Property name plus the string ``0000``.
+- PropertyPage: the address of the page to which updates are to be
+  written is the first 36 characters of the hash of the identifier of
+  its associated Record, plus the first 22 characters of the hash of
+  its Property name, plus the hex representation of the current_page
+  of its associated Property left-padded to length 4 with 0s.
+- Proposal: the first 36 characters of the hash of the identifier of
+  its associated Record plus the first 22 characters of its receiving
+  Agent plus the first 4 characters of the hash of its timestamp.
+- Record: the first 62 characters of the hash of its identifier.
+- Record Type: the first 62 characters of the hash of the name of the
+  type.
+
+For example, if 'fish-456' is a Record with a 'temperature' Property
+and a current_page of 28, the address for that PropertyPage is:
```

Comment:
Does current_page need to display in fixed-width font?  If so, double backticks.